common/lib/xmodule/xmodule/templates/about/overview.yaml

```diff
@@ -4,121 +4,121 @@ metadata:
-          </p>
-        </li>
-        <!-- 3 -->
-        <li class="course-about-list__item">
-          <div class="course-about-list__title">
-            <img src="/static/edx-theme/images/course-about-3.svg" alt="">
-            <span>Проводить мероприятия</span>
-          </div>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipiscing elit Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-          </p>
-        </li>
-        <!-- 4 -->
-        <li class="course-about-list__item">
-          <div class="course-about-list__title">
-            <img src="/static/edx-theme/images/course-about-4.svg" alt="">
-            <span>Оценивать результат</span>
-          </div>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipiscing elit Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-          </p>
-        </li>
-        <!-- 5 -->
-        <li class="course-about-list__item">
-          <div class="course-about-list__title">
-            <img src="/static/edx-theme/images/course-about-5.svg" alt="">
-            <span>Вовлекать аудиторию</span>
-          </div>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipiscing elit Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-          </p>
-        </li>
-        <!-- 6 -->
-        <li class="course-about-list__item">
-          <div class="course-about-list__title">
-            <img src="/static/edx-theme/images/course-about-6.svg" alt="">
-            <span>Формировать ядро общества</span>
-          </div>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipiscing elit Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-          </p>
-        </li>
-      </ul>
-    </section>
-  </div>
-  <section class="course-result course-page-section">
-    <h2 class="h2">Результат того стоит</h2>
-    <ul class="course-result-list">
-      <li class="course-result-list__item">
-        <div class="course-result-list__container">
-          <img class="course-result-list__image" src="/static/edx-theme/images/course-result-1.svg" alt="">
-          <span class="course-result-list__text">Выстроите развитие сообщества по прогнозам</span>
+  <section class="course-about course-page-section">
```

Comment:
Looks like this PR does more than just fixing the static. 
Commit description should probably be more descriptive.